lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.html

```diff
@@ -1,10 +1,12 @@
 <label class="adf-property-label"
+    [ngClass]="{'adf-property-label-not-editable' : !isEditable() && editable}"
```

Comment:
why there are 2 editable flags that look the same? can you have one check instead?